s1reader/s1_etad.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    rg_corr, az_corr = s1_etad.get_eta_correction_from_slc_burst(slc_burst,
                                                                         eta_dir,
                                                                         corr_type='bistatic',
                                                                         resample=True,
                                                                         unit='second')
</pre><p>Reducing the line length</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/opera-adt/notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done.</p>